src/plugins/intel_cpu/tests/unit/shape_inference_test/convolution_biased_shape_inference_test.cpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gmock/gmock.h>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "ov_ops/convolution.hpp"
+#include "utils.hpp"
+
+using namespace ov;
```

Comment:
Remove it.
There can be some collisions in type e.g. `Shape` is better to use ov:: for types from core.

Reply:
Done